app/src/main/res/values/strings.xml

```diff
@@ -158,16 +158,16 @@
     <string name="app_loyalty_card_keychain">Loyalty Card Keychain</string>
 
     <string name="privacy_policy">Privacy Policy</string>
-    <string name="privacy_policy_popup_text">Privacy policy notice (required by some app stores):\n\nWe collect NO DATA AT ALL, which anyone can confirm since our app is libre software.</string>
+    <string name="privacy_policy_popup_text">Privacy policy notice (required by some app stores):\n\nNO DATA IS COLLECTED AT ALL, which anyone can confirm since our app is libre software.</string>
     <string name="accept">Accept</string>
     <string name="importCatima">Import from Catima</string>
-    <string name="importCatimaMessage">Please select your Catima export file. It is most likely named Catima.csv.\n\nA Catima export file can be created by going to the Import/Export menu and pressing "Export".</string>
+    <string name="importCatimaMessage">Find a file likely named <i>Catima.csv</i> to import.\nOr create it from the Import/Export menu of another Catima app by pressing "Export" there first.</string>
     <string name="importFidme">Import from FidMe</string>
-    <string name="importFidmeMessage">Please select your FidMe export file. It is most likely named something like fidme-export-request-xxxxxx.zip.\n\nA FidMe export file can be created in the FidMe app by going to your profile, choosing "Data Protection" and then pressing "Extract my data".\n\nPlease note that FidMe does not store the barcode type in the export data so you will have to edit the imported cards manually.</string>
+    <string name="importFidmeMessage">Find a file likely named <i>fidme-export-request-xxxxxx.zip</i> to import, and then select the barcode types manually afterwards.\nOr create it from your FidMe profile by choosing "Data Protection" and then pressing "Extract my data" first.</string>
     <string name="importLoyaltyCardKeychain">Import from Loyalty Card Keychain</string>
-    <string name="importLoyaltyCardKeychainMessage">Please select your Loyalty Card Keychain export file. It is most likely named LoyaltyCardKeychain.csv.\n\nA Loyalty Card Keychain export file can be created by going to the Import/Export menu and pressing "Export".</string>
+    <string name="importLoyaltyCardKeychainMessage">Find a file most likely named <i>LoyaltyCardKeychain.csv</i> to import.\nOr create it from the Import/Export menu in Loyalty Card Keychain by pressing "Export" first.</string>
     <string name="importVoucherVault">Import from Voucher Vault</string>
-    <string name="importVoucherVaultMessage">Please select your Voucher Vault export file. It is most likely named vouchervault.json.\n\nA Voucher Vault export file can be created by pressing "Export".</string>
+    <string name="importVoucherVaultMessage">Find a file likely named <i>vouchervault.json</i> to import.\nOr create it by pressing "Export" in Voucher Vault first.</string>
     <string name="barcodeId">Barcode value</string>
```

Comment:
Not 100% this isn't "encoding" https://en.wikipedia.org/wiki/Code_128 or "value" https://www.keyence.com/ss/products/auto_id/barcode_lecture/basic/mechanism/

Reply:
It isn't the encoding, the encoding is called "Barcode type".

This is the data stored inside the barcode, if different from the "Card ID". It could also have the value "Same as Card ID".

It was introduced to fix https://github.com/TheLastProject/Catima/issues/90.

Reply:
I would call that "card number" and "barcode data", but if those are the names I suppose those are the names.
I didn't change anything here, so all good.